### PR TITLE
Separated results into its own partial, modified post route to includ…

### PR DESCRIPTION
**Pull request by chrononaut76**
…e optional item id

### CODE REVIEW

File: config/routes.rb
Comment (lauradumi) [inline config/routes.rb:1]: Could you remove lines 4 through 9 as they're just part of the boilerplate and not needed for our app

**Review by lauradumi — DISMISSED** [review]
Remove lines 4 through 9 in the routes.rb file as they're just part of the boilerplate and not needed for our app

**Comment by chrononaut76** [comment]
@lauradumi Done!

**Review by gabrieleli09 — APPROVED** [review]
LGTM

**Comment by gabrieleli09** [comment]
need it to be able to merge

**Pull request merged**